tests/random_test.py

```diff
@@ -2134,38 +2134,47 @@ def test_grad_of_prng_key(self):
 @jtu.with_config(jax_default_prng_impl='rbg')
 class LaxRandomWithRBGPRNGTest(LaxRandomTest):
   def seed_prng(self, seed):
-    return random.key(seed, impl='rbg')
+    return random.PRNGKey(seed, impl='rbg')
 
-  @skipIf(not config.jax_enable_custom_prng, 'relies on typed key arrays')
   def test_split_shape(self):
     key = self.seed_prng(73)
     keys = random.split(key, 10)
-    self.assertEqual(keys.shape, (10,))
+    self.assertEqual(keys.shape[:1], (10,))
```

Comment:
```suggestion
    self.assertEqual(keys.shape, (10, *key.shape))
```

Also could use a similar pattern in the updated tests below.

Reply:
Nice – thank you